src/bindings/python/docs/code_examples.md

```diff
@@ -0,0 +1,413 @@
+# Examples of OpenVINO:tm: Python API code
+<!-- Do we keep it as is? -->
+
+#### Prerequisites
+<!-- Do we keep it here? -->
+*To be added...*
+
+##### Enviroment
+<!-- Link to enviroment setup -->
+*To be added...*
+
+##### Building
+<!-- Link to building instructions -->
+*To be added...*
+
+### Different ways of extending OpenVINO:tm: Python API
+
+###### Before start: Project's naming conventions
+General guide:
+* Snake case is used across the code. That includes modules (`runtime`, `offline_transformations`), function names and arguments/varaibles (`async_infer`, `wait`, `path_to_xml`).
+* Exception to above rule is naming of classes. For example: `Core`, `InferRequest` and `AsyncInferQueue` start each word with a capital letter.
```

Comment:
In the line above you used the term "snake case". I think, that you could use the term "camel case" when describing the style for classes.